bitrise.yml

```diff
@@ -661,6 +661,24 @@ workflows:
     - deploy-to-bitrise-io@2: {}
     before_run:
     - prep_all
+    meta:
+      bitrise.io:
+        stack: osx-xcode-15.0.x
+        machine_type_id: g2-m1-max.5core
+  ui-tests-5:
+    steps:
+    - xcode-test@4:
+        inputs:
+        - destination: $DEFAULT_TEST_DEVICE
+        - test_repetition_mode: retry_on_failure
+        - maximum_test_repetitions: "2"
+        - scheme: StripeConnect Example
+        - test_plan: StripeConnectExample
```

Comment:
🤔 Interesting. It looks like PaymentSheet tests are broken into "shards", presumably so the tests can be run in parallel for faster CI times. Something we might want to consider when we start to write UI tests.